docs/providers/aws/guide/serverless.yml.md

```diff
@@ -132,6 +132,8 @@ provider:
   versionFunctions: false
   # Processor architecture: 'x86_64' or 'arm64' via Graviton2 (default: x86_64)
   architecture: x86_64
+  # Maximum retry attempts when an asynchronous invocation fails (between 0 and 2; default: 2)
+  maximumRetryAttempts: 1
```

Comment:
Actually this is not supported on `provider` level - it has to be set directly on a function and it's documented in `functions.md` file. It would be good to add it in this fine, but in `Functions` section.